jaeger-thrift/src/test/java/io/jaegertracing/thrift/internal/senders/UdpLazinessTest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020, The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.jaegertracing.thrift.internal.senders;
+
+import org.junit.Test;
+
+public class UdpLazinessTest {
+    @Test
+    public void testLazyInitializationOfAgent() {
+        new UdpSender("unknown-host." + Math.random(), 55555, 0);
```

Comment:
`.test` is a TLD that exists for this kind of situation. Anything under that TLD is guaranteed to not exist.

```suggestion
        new UdpSender("agent.acme.test", 55555, 0);
```